hazelcast/include/hazelcast/client/IMap.h

```diff
@@ -666,19 +666,19 @@ namespace hazelcast {
             * Notice that map EntryProcessor runs on the nodes. Because of that, same class should be implemented in java side
             * with same classId and factoryId.
             *
-            * @tparam EntryProcessor type of entry processor class
-            * @tparam ResultType that entry processor will return
+            * @param EntryProcessor type of entry processor class
```

Comment:
tparam stands for template parameter, doxygen interpertes these two differently and produces different output. These should stay as `tparam`.